### PR TITLE
Add Portuguese Brazilian translation

### PR DESCRIPTION
**Pull request by gHenriqueCarlos**

### CODE REVIEW

**Review by LESS14 — COMMENTED** [review]
There's no need to name it "ptbr" since, in this grammar, there's no distinction between European Portuguese and Brazilian Portuguese. For better understanding, rename it as "pt".

File: src/locales/translations/ptbr.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  nickname: "Apelido",
+  settings: "Configurações",
+  minimize: "Minimizar",
+  maximize: "Maximizar",
+  close: "Fechar",
+  add_server_modal_description_1: "Adicione servidores manualmente à lista de favoritos.",
+  add_server_modal_description_2: "Exemplo: 127.0.0.1:7777",
+  add: "Adicionar",
+  server: "Servidores",
+  address: "Endereço",
```
Comment (LESS14) [inline src/locales/translations/ptbr.ts]: Change this part to "Endereço IP".

File: src/locales/translations/ptbr.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    "Remover servidor selecionado dos favoritos",
+  add_selected_server_to_favorites: "Adicionar servidor selecionado aos favoritos",
+  add_server: "Adicionar Servidor",
+  hide_player_and_rule_list: "Ocultar lista de jogadores e regras",
+  show_player_and_rule_list: "Mostrar lista de jogadores e regras",
+  copy_server_info: "Copiar informações do servidor",
+  settings_gta_path_input_label:
+    "GTA: San Andreas caminho (onde o SA-MP está instalado)",
+  browse: "Selecionar",
+  settings_import_nickname_gta_path_from_samp:
```
Comment (LESS14) [inline src/locales/translations/ptbr.ts]: Change this part to "Importar configurações do SA-MP"

**Comment by pkfln** [comment]
> There's no need to name it "ptbr" since, in this grammar, there's no distinction between European Portuguese and Brazilian Portuguese. For better understanding, rename it as "pt".

There definitely are differences. The correct locale is pt-BR for this one.

File: src/locales/translations/ptbr.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "Importar apelido e caminho gtasa das configurações do SA-MP",
+  settings_import_samp_favorite_list: "Importar lista de favoritos do SA-MP",
+  settings_reset_application_data:
+    "Redefinir dados do aplicativo (limpar configurações e listas)",
+  settings_new_update_available: "⚠ Nova atualização disponível. Clique para fazer o download! ⚠",
```
Comment (LESS14) [inline src/locales/translations/ptbr.ts]: Change this part to "⚠ Nova atualização disponível. Clique aqui para baixar a atualização! ⚠"

File: src/locales/translations/ptbr.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  settings_credits_made_by: "Feito com ❤️ por",
+  settings_credits_view_source_on_github: "Veja o código-fonte no GitHub",
```
Comment (LESS14) [inline src/locales/translations/ptbr.ts]: Change this part to "Código fonte disponível no GitHub"

File: src/locales/translations/ptbr.ts [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  update_modal_update_available_title: "Atualização disponível!",
+  update_modal_update_available_description:
+    'Nova versão do launcher disponível!\nVersão do seu launcher: {{ version }}\nVersão atual do launcher: {{ newVersion }}\nClique em "Baixar" para abrir a página de downloads',
+  download: "Baixar",
+  update_modal_remind_me_next_time: "Me lembre na próxima vez",
```
Comment (LESS14) [inline src/locales/translations/ptbr.ts]: Change this part to "Me lembre mais tarde"

**Review by LESS14 — CHANGES_REQUESTED** [review]

**Comment by LESS14** [comment]
> > There's no need to name it "ptbr" since, in this grammar, there's no distinction between European Portuguese and Brazilian Portuguese. For better understanding, rename it as "pt".
> 
> There definitely are differences between European and Brazilian Portuguese. The correct locale is pt-BR for this one.

Not in this case, since this grammar can be easily understood by both accents.

**Comment by pkfln** [comment]
I just skimmed through it and as someone that learned European Portuguese, I wouldn't have known what they meant by "Apelido" in this case. I've only seen username/nickname/alcunha/usario being used.

**Comment by gHenriqueCarlos** [comment]
> I just skimmed through it and as someone that learned European Portuguese, I wouldn't have known what they meant by "Apelido" in this case. I've only seen username/nickname/alcunha/usario being used.

We can replace 'Apelido' with 'Nome de Usuário' or just 'Usuário.' I believe it will work well for both. What do you think?

**Comment by LESS14** [comment]
> We can replace 'Apelido' with 'Nome de Usuário' or just 'Usuário.' I believe it will work well for both. What do you think?

It would be better, in my opinion, since it would make it easier for both European Portuguese and Brazilian Portuguese to understand.

**Review by LESS14 — APPROVED** [review]

**Review by pushline — APPROVED** [review]

**Review by Southclaws — APPROVED** [review]
Muito bom, obrigado!

**Pull request merged**